boot/zephyr/targets/nrf52840_pca10056.h

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (C) 2017, Linaro Ltd
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_TARGETS_NRF52840_PCA10056_
+#define H_TARGETS_NRF52840_PCA10056_
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
+#define FLASH_AREA_IMAGE_0_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_1_OFFSET	0x82000
+#define FLASH_AREA_IMAGE_1_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_SCRATCH_OFFSET	0xfc000
+#define FLASH_AREA_IMAGE_SCRATCH_SIZE	0x01000
```

Comment:
Hmm, by that argument we already have a problem on Nitrogen, which we need to fix as well.

This is the current situation on Nitrogen:

|                 | Start   | End     | Size (KiB) | User           |
|-----------------|---------|---------|------------|----------------|
| Sectors 0-7     | 0x00000 | 0x07FFF | 32         | mcuboot        |
| Sectors 8-65    | 0x08000 | 0x41FFF | 232        | Image 0        |
| Sectors 66-123  | 0x42000 | 0x7BFFF | 232        | Image 1        |
| Sector 124      | 0x7C000 | 0x7CFFF | 4          | Scratch        |
| Sectors 125-127 | 0x7D000 | 0x7FFFF | 12         | FOTA app state |

At 10,000 erases (which is the figure for the NRF52832 on that board), we get 172 updates before we expect wear on the part. I expect we are close to seeing this on some of our boards, e.g. the ones @embeddedandroid uses for CI.

Subtracting mcuboot and the FOTA app state, we have 117 4-KiB sectors to divide between images and scratch on Nitrogen, and 245 sectors on NRF52840.

For Nitrogen, splitting up 117 sectors into images and a scratch partition sized 1/4 that of the images would be 52 KiB (13 sectors) scratch, 208 KiB (52 sectors) images.

For NRF52840, we'd get 116 KiB (29 sectors) 432 KiB (108 sectors) images.

Seem reasonable to everyone? I'll make the changes if so.